avocado/utils/config_change_validation.py

```diff
@@ -0,0 +1,205 @@
+"""
+
+SUMMARY
+------------------------------------------------------
+Config change validation: Utility for testing config file changes.
+
+Copyright: 2012 Intra2net AG
+
+
+INTERFACE
+------------------------------------------------------
+
+"""
+
+import os
+import shutil
+
+# avocado utilities
+from avocado.utils import process
+
+
+def get_temp_file_path(file_path):
+    """
+    Generates a temporary filename.
+
+    :param str file_path: file path prefix
+    :returns: appended file path
+    :rtype: str
+    """
+    return file_path + '.tmp'
+
+
+def make_temp_file_copies(file_paths):
+    """
+    Creates temporary copies of the provided files.
+
+    :param file_paths: copied file paths
```

Comment:
Should this be the files *to be* copied (and not already copied) ?

Reply:
Correct, resolved on the next push.